x/examples/fetch-psiphon/README.md

```diff
@@ -5,7 +5,7 @@ This fetch tool illustrates how to use Psiphon as a stream dialer.
 Usage:
 
 ```sh
-go run github.com/Jigsaw-Code/outline-sdk/x/examples/fetch-psiphon@latest -config config.json https://ipinfo.io
+go run -C ./x/examples/fetch-psiphon -tags psiphon . -config config.json https://ipinfo.io
```

Comment:
Please don't use the local repo. This forces people to clone the repo. We prefer the absolute way, which lets peopl run it from anywhere without explicit cloning.